ocis/pkg/init/init.go

```diff
@@ -289,6 +294,10 @@ func CreateConfig(insecure, forceOverwrite bool, configPath, adminPassword strin
 	if err != nil {
 		return fmt.Errorf("could not generate random password for tokenmanager: %s", err)
 	}
+	collaborationJwtSecret, err := generators.GenerateRandomPassword(passwordLength)
```

Comment:
Just saying that the original code used the uniuri lib, which used alphanumeric characters. Current code will also include additional symbols. I don't know if they could cause issues

Reply:
Good question!